adapters/nextmillennium/nextmillennium.go

```diff
@@ -99,12 +99,22 @@ func (adapter *adapter) buildAdapterRequest(prebidBidRequest *openrtb2.BidReques
 }
 
 func createBidRequest(prebidBidRequest *openrtb2.BidRequest, params *openrtb_ext.ImpExtNextMillennium) *NextMillenniumBidRequest {
+	placementID := params.PlacementID
 	bidRequest := NextMillenniumBidRequest{
 		ID:   prebidBidRequest.ID,
 		Test: uint8(prebidBidRequest.Test),
 	}
-	bidRequest.Ext.Prebid.StoredRequest.ID = params.PlacementID
 
+	if params.GroupID != nil {
```

Comment:
Please add JSON test cases for the changes in this PR.

Reply:
@mansinahar Thank you for reply.
I have added JSON-test files and made changes in the code

Reply:
Thanks @mifanich! I will re-review this. Can you please also address the other comment below that I missed earlier when you get a chance?

Reply:
Done!
Changed to string instead of pointer.